src/test/resources/db/migration/R__test_data.sql

```diff
@@ -1,32 +1,51 @@
+DELETE from course_audience;
```

Comment:
Not quite sure I understand what this commit is doing from the commit message:

> Updated latest src/main (V24__) and src/test (R__) flyway migrations for consistency.

Could we add a bit more detail about what we mean by "for consistency"?

Reply:
Ah I've just read the NB of the PR description. Could we add that to the commit message please?